### ISSUE TITLE
[Docs] Is it possible to train 3D Grounding directly without loading the 3DDet model?

### ISSUE THREAD

**Issue by Fan123456-c** [OpenRobotLab/EmbodiedScan#91]
### Branch

main branch  https://mmdetection3d.readthedocs.io/en/latest/

### 📚 The doc issue

In the 3D Grounding task, your model loads a pre-trained 3Ddet model. When I don't load it, the accuracy of the 3D Grounding task decreases. Has the author tried training 3D Grounding directly without loading the 3Ddet model? If so, could you provide some training parameter suggestions such as lr, train_epoch, MultiStepLR?
![image](https://github.com/user-attachments/assets/5bab37c6-62a8-46e8-a367-8a1d883f9d25)

**Comment by Tai-Wang** [comment]
Yes, loading the checkpoint pretrained on the 3D detection task effectively accelerates the convergence of training grounding models. You may try to train from scratch but need a much longer training time to obtain a similar performance based on our experiences (and previous papers).
